examples/YOLOv8-Action-Recognition/readme.md

```diff
@@ -1,116 +0,0 @@
-# Zero-shot Action Recognition with YOLOv8 (Inference on Video)
-
-- Action recognition is a technique used to identify and classify actions performed by individuals in a video. This process enables more advanced analyses when multiple actions are considered. The actions can be detected and classified in real time.
-- The system can be customized to recognize specific actions based on the user's preferences and requirements.
-
-## Table of Contents
-
-- [Step 1: Install the Required Libraries](#step-1-install-the-required-libraries)
-- [Step 2: Run the Action Recognition Using Ultralytics YOLOv8](#step-2-run-the-action-recognition-using-ultralytics-yolov8)
-- [Usage Options](#usage-options)
-- [FAQ](#faq)
-
-## Step 1: Install the Required Libraries
-
-Clone the repository, install dependencies and `cd` to this local directory for commands in Step 2.
-
-```bash
-# Clone ultralytics repo
-git clone https://github.com/ultralytics/ultralytics
-
-# cd to local directory
-cd examples/YOLOv8-Action-Recognition
-
-# Install dependencies
-pip install -U -r requirements.txt
-```
-
-## Step 2: Run the Action Recognition Using Ultralytics YOLOv8
-
-Here are the basic commands for running the inference:
-
-### Note
-
-The action recognition model will automatically detect and track people in the video, and classify their actions based on the specified labels. The results will be displayed in real-time on the video output. You can customize the action labels by modifying the `--labels` argument when running the script.
```

Comment:
This could just be me, but I wonder if it might be better to avoid using "classify" so there's no confusion about the `classify` task? Maybe "categorize their actions", "label their actions", or "identify their actions"?

Reply:
`action recognition` is a subbranch of `video classification` in the literature. Most of the time, these words are used interchangeably. However, product-wise, your suggestions might fit better @Burhan-Q.